### PR TITLE
Added seeds.rb to populate the DB

### PR DESCRIPTION
**Pull request by jayk-u**
Worked on my end, should be the same for you. Please review, thank you ❤️

### CODE REVIEW

File: db/seeds.rb [hunk 1]
```diff
@@ -5,3 +5,37 @@
...
+Game.destroy_all
+User.destroy_all
+Reservation.destroy_all
+Review.destroy_all
+
+puts "Creating database..."
+
+puts "Adding users..."
+
+User.create!(email: "pierre@games.com", password: "123456", first_name: "Pierre", last_name: "Bodin", location: "Paris")
```
Comment (DumontNicolas) [inline db/seeds.rb]: Pierre = x

File: db/seeds.rb [hunk 2]
```diff
@@ -5,3 +5,37 @@
...
+p "Nico added!"
+User.create!(email: "clara@games.com", password: "123456", first_name: "Clara", last_name: "Heurteux", location: "Paris")
+p "Clara added!"
+User.create!(email: "jay@games.com", password: "123456", first_name: "Jay", last_name: "Cormier", location: "Rennes")
+p "Jay added!"
+p "Everyone's here."
+
+p "Creating games..."
+
+Game.create!(name: "Nemesis", description: "A sci-fi cooperation game... with a traitor", user_id: User.all.sample.id, category: "Coop", price: 15)
```
Comment (DumontNicolas) [inline db/seeds.rb]: Nemesis = x

File: db/seeds.rb [hunk 3]
```diff
@@ -5,3 +5,37 @@
...
+Game.create!(name: "Azul", description: "Be the first to build the perfect canvas for the king", user_id: User.all.sample.id, category: "Economy", price: 10)
+Game.create!(name: "The Crew", description: "Your team crashed on a planet and you need to leave asap... Unless the planet manages to keep you there", user_id: User.all.sample.id, category: "Coop", price: 10)
+Game.create!(name: "Gloomhaven", description: "Build a team and explore dungeons in this cooperative campaign-oriented game", user_id: User.all.sample.id, category: "Adventure", price: 20)
+
+p "4 games have been added."
+p "And now, reservations..."
+
+Reservation.create!(start_date: "10 Mar 2021", end_date: "12 Mar 2021", user: User.all.sample, game: Game.all.sample, status: "confirmed", total: 45)
+Reservation.create!(start_date: "15 Mar 2021", end_date: "17 Mar 2021", user: User.all.sample, game: Game.all.sample, status: "pending", total: 30)
+p "2 reservations have been created"
```
Comment (DumontNicolas) [inline db/seeds.rb:42]: Pierre.games << Nemesis
Nemesis.reviews << review
ect

**Comment by jayk-u** [comment]
@DumontNicolas Should be fixed now, could you please have a look?

**Review by DumontNicolas — APPROVED** [review]

**Pull request merged**